api/handler.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+from api import get_co2_estimate
+from transport_co2 import Mode
+
+def strtofloat(val):
+    if isinstance(val, str):
+        val = float(val)
+    return val
+
+def post(event, context):
+    try:
+        vars = event.get("queryStringParameters", dict())
+        if vars != None:
+            print(f'estimate-co2 sourceIp={event["requestContext"]["identity"]["sourceIp"]} userAgent={event["requestContext"]["identity"]["userAgent"]}, input={vars}')
+            
+            transport_mode = vars.get("transport_mode")
+            if isinstance(transport_mode, str):
+                transport_mode = Mode[transport_mode.upper()]
+
+            distance_km = strtofloat(vars.get("distance_km"))
+            vehicle_occupancy = strtofloat(vars.get("vehicle_occupancy"))
+            origin_lat = strtofloat(vars.get("origin_lat"))
+            origin_lon = strtofloat(vars.get("origin_lon"))
+            destination_lat = strtofloat(vars.get("destination_lat"))
+            destination_lon = strtofloat(vars.get("destination_lon"))
+
+            estimate = get_co2_estimate(
+                transport_mode = transport_mode,
+                distance_km = distance_km,
+                vehicle_occupancy = vehicle_occupancy, 
+                origin_lat = origin_lat, 
+                origin_lon = origin_lon,
+                destination_lat = destination_lat, 
+                destination_lon = destination_lon
+            )
+
+            if estimate.get("error") == None:
```

Comment:
Similar to above, we could do this:

```python
if estimate.get("error"):
                response = {
                    "statusCode": 400,
                    "body": json.dumps({
                        "error:": estimate.get("error")
                    })
                }
                return response
```

and then unindent the rest by one level.

Reply:
done